woocommerce/woocommerce-gateway-stripe
# fix: save section position

## Changes proposed in this Pull Request:

I noticed a little issue on the "save" section, it looks a bit too cramped. By reorganizing a bit the code, it can be fixed.

# Testing instructions
- Ensure you have the `_wcstripe_feature_upe_settings` option set to `'1'`
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe
- "Save" button section is updated

Before:
![Screen Shot 2021-08-25 at 10 39 37 AM](https://user-images.githubusercontent.com/273592/130821798-2ee25dd5-978b-418f-91a1-2547f15eb6a4.png)

After:
![Screen Shot 2021-08-25 at 10 40 10 AM](https://user-images.githubusercontent.com/273592/130821822-4b92c64a-d67a-485f-bc0f-173fbc830ca2.png)
